server/routes/api/place.py

```diff
@@ -113,6 +114,56 @@ def child(dcid):
     return json.dumps(child_places)
 
 
+# Defines the map from higher geos to their respective subgeos.
+LEVEL_MAP = {
+    "Country": "AdministrativeArea1",
+    "AdministrativeArea1": "AdministrativeArea2",
+    "AdministrativeArea2": "City"
+}
+@bp.route('child/statvars')
+@cache.memoize(timeout=3600 * 24)  # Cache for one day.
+def child_statvars():
+    """
+    Gets all statistical variable avaiable for a particular sublevel of a dcid. 
+
+    API Params:
+        dcid -> The place dcid geoID to information for, as a string.
+        level -> The level of children to pull for, as a string.
+    Example Query:
+        api/place/child/statvars?dcid=country/USA&level=State
+        Returns all statistical variables that are value for states of the USA.
+    Returns:
+        A json list of all the available statistical variables.
+    """
+    # Get required params.
+    requested_dcid = request.args.get("dcid")
+    if requested_dcid is None:
+        return jsonify({"error": "Must provide a 'dcid' field!"}, 400)
+    requested_level = request.args.get("level")
+    if requested_level is None:
+        requested_dcid_type = dc.get_property_values([requested_dcid],
+                                                "typeOf")[requested_dcid]
+        # TODO(iancostello): Handle a failed function call, e.g., returns None.
+        # TODO(iancostello): Handle the case where display_level is None.
```

Comment:
```suggestion
        # TODO(iancostello): Handle the case where requested_level is None.
```

???  I thought this was handled above.